htdocs/adherents/card.php

```diff
@@ -634,6 +634,12 @@
 					$msg     = $arraydefaultmessage->content;
 				}
 
+                if (empty($labeltouse)) {
+                    //fallback on the old configuration.
```

Comment:
Ok do I keep the fallback too ?

Reply:
I put an error message since the action cannot continue.
For me  : 
* Success when the requested action has been done.
* Warning when something is wrong but the system accomplished the action 
* Error when the action cannot be performed.